lib/wallet/integration/src/Test/Integration/Scenario/API/Byron/Transactions.hs

```diff
@@ -353,18 +354,142 @@ spec = describe "BYRON_TRANSACTIONS" $ do
                     (.>= Quantity (faucetAmt - faucetUtxoAmt))
             ]
 
-        eventually "wa and wb balances are as expected" $ do
-            rb <- request @ApiWallet ctx
+        let txId = getFromResponse (#id) r
+        eventually "wa and wb balances are as expected as well as fees on txs" $ do
+            -- check that tx and balance on source Byron wallet is as expected
+            rTxSrc <- request @(ApiTransaction n) ctx
+                (Link.getTransaction @'Byron wByron (ApiTxId txId)) Default Empty
+            verify rTxSrc
+                [ expectSuccess
+                , expectField (#amount . #getQuantity) (`shouldBe` amt + feeEstMax)
+                , expectField (#fee . #getQuantity) (`shouldBe` feeEstMax)
+                , expectField (#direction . #getApiT) (`shouldBe` Outgoing)
+                ]
+            rSrc <- request @ApiByronWallet ctx
+                (Link.getWallet @'Byron wByron) Default Empty
+            verify rSrc
+                [ expectSuccess
+                , expectField
+                        (#balance . #available)
+                        (`shouldBe` Quantity (faucetAmt - feeEstMax - amt))
```

Comment:
```suggestion
                , expectField (#balance . #total . #getQuantity) $
                    between
                        ( faucetAmt - feeEstMax - amt
                        , faucetAmt - feeEstMin - amt
                        )
```

Same issue as above with fee estimation. Also, we now expect the `total` balance to have decreased — the `available` balance takes pending transactions into account, while the `total` balance only considers transactions that have been included in the ledger (though they can still be rolled back).